presto-main/src/test/java/com/facebook/presto/type/TestTypeRegistry.java

```diff
@@ -250,6 +259,22 @@ public void testCastOperatorsExistForCoercions()
         }
     }
 
+    @Test
+    public void testOperatorsImplemented()
+    {
+        for (Type type : typeRegistry.getTypes()) {
+            if (type.isComparable()) {
+                Stream.of(IS_DISTINCT_FROM, EQUAL, NOT_EQUAL)
+                        .forEach(operatorType -> functionRegistry.resolveOperator(operatorType, ImmutableList.of(type, type)));
```

Comment:
What about putting each operator on its own line, like this:

```
functionRegistry.resolveOperator(EQUAL, ImmutableList.of(type, type));
functionRegistry.resolveOperator(NOT_EQUAL, ImmutableList.of(type, type));
functionRegistry.resolveOperator(IS_DISTINCT_FROM, ImmutableList.of(type, type))
```

I think the repetition would be pretty minimal. And it will make it easier for readers to understand what is missing when this test fails.

ditto for orderable operators.